ingestor/service.go

```diff
@@ -372,6 +377,15 @@ func (s *Service) HandleTransfer(w http.ResponseWriter, r *http.Request) {
 		}
 	}()
 
+	for _, prefix := range s.dropFilePrefixes {
+		if strings.HasPrefix(filename, prefix) {
+			logger.Errorf("Transfer request rejected due to prefix %q: %q", prefix, filename)
+			m.WithLabelValues(strconv.Itoa(http.StatusBadRequest)).Inc()
+			http.Error(w, "filename is configured to be dropped", http.StatusBadRequest)
```

Comment:
Wondering if we should just silently drop this instead.  e.g. just return a 200 that we accepted and move on.

Reply:
I think we should probably do that. Drop implies we're accepting it but not forwarding it on, but not returning an error either.